README.md

```diff
@@ -130,6 +130,27 @@ end
 The `setup` is run once on each worker process that requires it;
 it is not run before every `@testitem` that depends on the setup.
 
+#### Skipping tests
+
+The `skip` keyword can be used to skip a `@testitem`, meaning no code inside that test-item will run.
+A skipped test-item logs that it is being skipped and records a single "skipped" test result, similar to `@test_skip`.
+
+```julia
+@testitem "skipped" skip=true begin
+    @test false
+end
+```
+
+If `skip` is an given an `Expr`, it must return a `Bool` indicating whether or not to skip the test-item.
+This expression will be run in a new module similar to a test-item immediately before the test-item would be run.
+
+```julia
+# Don't run "orc v1" tests if we don't have orc v1
+@testitem "orc v1" skip=:(using LLVM; !LLVM.has_orc_v1()) begin
+    # tests
+end
+```
```

Comment:
It still seems like this could just be `tag=:orc_v1`, and then in the runtests invocation you write
```julia
if LLVM.has_orc_v1()
    runtests(label=:orc_v1)
end
```
?

But I guess maybe it's nicer to have the skip logic at the testitem's definition?